openbb_terminal/stocks/tradinghours/pandas_market_cal_view.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import pandas as pd
+
+
+from openbb_terminal.decorators import log_start_end
+from openbb_terminal.stocks.tradinghours import pandas_market_cal_model
+from openbb_terminal.helper_funcs import print_rich_table
+from openbb_terminal.rich_config import console
+
+logger = logging.getLogger(__name__)
+
+
+@log_start_end(log=logger)
+def get_all_holiday_exchange_short_names() -> pd.DataFrame:
+    """Get all holiday exchanges short names.
+
+    Parameters
+    ----------
+
+    Returns
+    -------
+    pd.DataFrame
+        All available exchange with holiday data short names
+    """
+
+    return pandas_market_cal_model.get_all_holiday_exchange_short_names()
+
+
+@log_start_end(log=logger)
+def display_exchange_holidays(exchange_symbol: str, year: int = 2022):
+    """Display current exchange holiday calendar.
+
+    Parameters
+    ----------
+    symbol : str
+        Exchange symbol
+    year : int
+        Calendar year
+    """
+    exchange = pandas_market_cal_model.get_exchange_holidays(exchange_symbol, year)
+
+    if len(exchange) == 0 or exchange.empty:
+        console.print(
+            "[red]"
+            + "No exchange data loaded.\n"
+            + "Make sure you picked a valid exchange symbol and year."
+            + "[/red]\n"
+        )
+        return
+
+    print_rich_table(
+        exchange,
+        show_index=True,
```

Comment:
We should have show_index=False.  The index is kind of confusing in this context:
```
         NYSE          
┏━━━━━━┳━━━━━━━━━━━━━━┓
┃      ┃ Holiday Date ┃
┡━━━━━━╇━━━━━━━━━━━━━━┩
│ 1788 │ 2022-01-17   │
├──────┼──────────────┤
```

Reply:
thanks for the feedback. Since this process is new to me should I update on my fork and them submit a new pull request will for all the same files in the original?

Reply:
Hey!  If you keep on the same branch that you did this work on, you can just make the changes and then commit/push and it will update!

Reply:
I have committed this change now.

Reply:
It is still showing show_index=True.  Make sure to push your changes!

Reply:
sorry, I have pushed to my forked copy now. Still getting the hang of Git and Github